app/services/league/controllers/leagueController.js

```diff
@@ -88,6 +114,14 @@ exports.leaveLeague = async (req, res) => {
                 if (err) throw err;
             },
         ),
+        await League.findOneAndUpdate(
+            { 'portfolioList.owner': username },
+            { $pull: { portfolioList: { owner: username } } },
+            { new: true },
+            (err) => {
+                if (err) throw err;
+            },
+        ),
```

Comment:
Combine with above request from lines 101 to 108

Reply:
I don't know if this is possible